### PR TITLE
Add DAST findings

### PR DESCRIPTION
**Pull request by mahmoudadly-cobalt**
https://zombie.atlassian.net/browse/FX-4418

(wait for https://github.com/cobalthq/cobalt-api-docs/pull/242 to be merged to have the page weights set properly)

### CODE REVIEW

File: versions/v2/content/dast-findings.md [hunk 1]
```diff
@@ -0,0 +1,143 @@
...
+| `cursor`   | N/A     | Used for [pagination](./#pagination). Example: `https://api.us.cobalt.io/dast/findings?cursor=a1b2c3d4` |
+| `limit`    | `10`    | If specified, returns only a specified amount of findings. Example: `https://api.us.cobalt.io/dast/findings?limit=5` |
+| `target`   | N/A     | If specified, returns findings scoped to this target id. Example: `https://api.us.cobalt.io/dast/findings?target=dt_GZgcehapJUNh6mjNuqsE4T` |
+| `scan`     | N/A     | If specified, returns findings scoped to this scan id. Example: `https://api.us.cobalt.io/dast/findings?scan=dsc_GZgcehapJUNh6mjNuqsE4T` |
+
+### Response Fields
+
+| Field           | Description                                                                         |
+|-----------------|-------------------------------------------------------------------------------------|
+| `id`                | A unique ID representing the DAST scan. Starts with `dsc_` |
```
Comment (grayson-cobalt) [inline versions/v2/content/dast-findings.md:72]:
```suggestion
| `id`                | A unique ID representing the DAST finding. Starts with `dsc_` |
```

File: versions/v2/content/dast-findings.md [hunk 2]
```diff
@@ -0,0 +1,143 @@
...
+
+### HTTP Request
+
+`GET https://api.us.cobalt.io/dast/findings/YOUR-DAST-FINDING-IDENTIFIER`
+
+### Response Fields
+
+| Field           | Description                                                                         |
+|-----------------|-------------------------------------------------------------------------------------|
+| `id`                | A unique ID representing the DAST scan. Starts with `dsc_` |
```
Comment (grayson-cobalt) [inline versions/v2/content/dast-findings.md:132]:
```suggestion
| `id`                | A unique ID representing the DAST finding. Starts with `dsc_` |
```

File: versions/v2/content/dast-findings.md [hunk 3]
```diff
@@ -0,0 +1,143 @@
...
+| `cursor`   | N/A     | Used for [pagination](./#pagination). Example: `https://api.us.cobalt.io/dast/findings?cursor=a1b2c3d4` |
+| `limit`    | `10`    | If specified, returns only a specified amount of findings. Example: `https://api.us.cobalt.io/dast/findings?limit=5` |
+| `target`   | N/A     | If specified, returns findings scoped to this target id. Example: `https://api.us.cobalt.io/dast/findings?target=dt_GZgcehapJUNh6mjNuqsE4T` |
+| `scan`     | N/A     | If specified, returns findings scoped to this scan id. Example: `https://api.us.cobalt.io/dast/findings?scan=dsc_GZgcehapJUNh6mjNuqsE4T` |
+
+### Response Fields
+
+| Field           | Description                                                                         |
+|-----------------|-------------------------------------------------------------------------------------|
+| `id`                | A unique ID representing the DAST finding. Starts with `dsc_` |
```
Comment (grayson-cobalt) [inline versions/v2/content/dast-findings.md:72]:
```suggestion
| `id`                | A unique ID representing the DAST finding. Starts with `dfi_` |
```

File: versions/v2/content/dast-findings.md [hunk 4]
```diff
@@ -0,0 +1,143 @@
...
+
+### HTTP Request
+
+`GET https://api.us.cobalt.io/dast/findings/YOUR-DAST-FINDING-IDENTIFIER`
+
+### Response Fields
+
+| Field           | Description                                                                         |
+|-----------------|-------------------------------------------------------------------------------------|
+| `id`                | A unique ID representing the DAST finding. Starts with `dsc_` |
```
Comment (grayson-cobalt) [inline versions/v2/content/dast-findings.md:132]:
```suggestion
| `id`                | A unique ID representing the DAST finding. Starts with `dfi_` |
```

File: versions/v2/content/dast-findings.md (hunk 3 above)
Comment (mahmoudadly-cobalt) [reply versions/v2/content/dast-findings.md]: oops! sorry

File: versions/v2/content/dast-findings.md [hunk 5]
```diff
@@ -0,0 +1,143 @@
...
+| `id`                | A unique ID representing the DAST finding. Starts with `dfi_` |
+| `target_id`         | A unique ID representing the DAST target. Starts with `dt_` |
+| `scan_ids`          | An array of unique ID representing the scans that originated the vulnerability finding. Starts with `dsc_` |
+| `title`             | Name of the vulnerability |
+| `last_found_at`     | Date and time of when the vulnerability was last found, in ISO 8601 UTC format. |
+| `severity`          | Severity of the vulnerability finding: `10` is low. `20` is medium. `30` is high. |
+| `state`             | State of the vulnerability finding: [`notfixed`, `invalid`, `accepted`, `fixed`] |
+| `affected_url`      | URL of the vulnerability finding |
+| `description`       | Description of the vulnerability. |
+| `proof_of_concept`  | Evidence with proof of the vulnerability finding. |
```
Comment (grayson-cobalt) [inline versions/v2/content/dast-findings.md:81]:
```suggestion
| `proof_of_concept`  | Evidence of the vulnerability finding. |
```

File: versions/v2/content/dast-findings.md [hunk 6]
```diff
@@ -0,0 +1,143 @@
...
+| Field           | Description                                                                         |
+|-----------------|-------------------------------------------------------------------------------------|
+| `id`                | A unique ID representing the DAST finding. Starts with `dfi_` |
+| `target_id`         | A unique ID representing the DAST target. Starts with `dt_` |
+| `scan_ids`          | An array of unique ID representing the scans that originated the vulnerability finding. Starts with `dsc_` |
+| `title`             | Name of the vulnerability |
+| `last_found_at`     | Date and time of when the vulnerability was last found, in ISO 8601 UTC format. |
+| `severity`          | Severity of the vulnerability finding: `10` is low. `20` is medium. `30` is high. |
+| `state`             | State of the vulnerability finding: [`notfixed`, `invalid`, `accepted`, `fixed`] |
+| `affected_url`      | URL of the vulnerability finding |
```
Comment (grayson-cobalt) [inline versions/v2/content/dast-findings.md:139]:
```suggestion
| `affected_url`      | URL affected by the found vulnerability |
```

**Review by grayson-cobalt — APPROVED** [review]
Thank you!

**Pull request merged**